pyEpiabm/pyEpiabm/sweep/demographics_sweep.py

```diff
@@ -0,0 +1,99 @@
+#
+# Sweep to record demographic information of the population in a .csv file
+#
+
+import os
+import typing
+
+from pyEpiabm.output import _CsvDictWriter
+
+from .abstract_sweep import AbstractSweep
+
+
+class DemographicsSweep(AbstractSweep):
+    """Class to sweep through the population at the beginning of the simulation
+    and record their demographic information
+
+    """
+
+    def __init__(self, file_params: typing.Dict):
+        """Initiate file parameters referring to the file location and
+        which columns are to be included
+
+        file_params Contains:
+            * `output_dir`: String for the location of the output file, \
+               as a relative path
+            * `spatial_output`: Boolean to determine whether a spatial output \
+               should be used
+            * `age_output`: Boolean to determine whether the output will \
+                have ages
+
+        """
+        if "output_dir" not in file_params:
+            raise ValueError("output_dir must be specified in file_params")
+        self.spatial_output = file_params["spatial_output"] \
+            if "spatial_output" in file_params else False
+        self.age_output = file_params["age_output"] \
+            if "age_output" in file_params else False
+
+        # Here we set up the writer
+        folder = os.path.join(os.getcwd(),
+                              file_params["output_dir"])
+        file_name = "demographics.csv"
```

Comment:
Just confer with the other group - they were expecting it all in one file. This doesn't duplicate data, as we only have one column per person anyway. But I am also very happy for you to keep two separate files (this actually might be better as we may well rerun the sim on the same population), so just let them know about this!

Reply:
I don't think they were expecting it in one file, when we chatted to them we let them know that we would have 3 separate files: one for demographics, one for infection history with status data and one for infection history with infectiousness data. But I will let them know again just in case